_posts/2020-06-29-release-1.11.0.md

```diff
@@ -0,0 +1,299 @@
+
+## New Features and Improvements
+
+### Unaligned Checkpoints (Beta)
+
+Triggering a checkpoint in Flink will cause a [checkpoint barrier]({{ site.DOCS_BASE_URL }}flink-docs-release-1.11/internals/stream_checkpointing.html#barriers) to flow from the sources of your topology all the way towards the sinks. For operators that receive more than one input stream, the barriers flowing through each channel need to be aligned before the operator can snapshot its state and forward the checkpoint barrier — typically, this alignment will take just a few milliseconds to complete, but it can become a bottleneck in backpressured pipelines as:
+
+ * Checkpoint barriers will flow much slower through backpressured channels, effectively blocking the remaining channels and their upstream operators during checkpointing;
+
+ * Slow checkpoint barrier propagation leads to longer checkpointing times and can, worst case, result in little to no progress in the application.
+
+To improve the performance of checkpointing under backpressure scenarios, the community is rolling out the first iteration of unaligned checkpoints ([FLIP-76](https://cwiki.apache.org/confluence/display/FLINK/FLIP-76%3A+Unaligned+Checkpoints)) with Flink 1.11. Compared to the original checkpointing mechanism (Fig. 1), this approach doesn’t wait for barrier alignment across input channels, instead allowing barriers to overtake in-flight records and forwarding them downstream before the synchronous part of the checkpoint takes place (Fig. 2).
+
+<div style="line-height:60%;">
+    <br>
+</div>
+
+<div class="row">
+  <div class="col-lg-6">
+    <div class="text-center">
+      <figure>
+		<img src="{{ site.baseurl }}/img/blog/2020-06-29-release-1.11.0/image1.gif" width="600px" alt="Aligned Checkpoints"/>
+		<br/><br/>
+		<figcaption><i><b>Fig.1:</b> Aligned Checkpoints</i></figcaption>
+	  </figure>
+    </div>
+  </div>
+  <div class="col-lg-6">
+    <div class="text-center">
+      <figure>
+		<img src="{{ site.baseurl }}/img/blog/2020-06-29-release-1.11.0/image2.png" width="600px" alt="Unaligned Checkpoints"/>
+		<br/><br/>
+		<figcaption><i><b>Fig.2:</b> Unaligned Checkpoints</i></figcaption>
+	  </figure>
+    </div>
+  </div>
+</div>
+
+<div style="line-height:150%;">
+    <br>
+</div>
+
+Because in-flight records have to be persisted as part of the snapshot, unaligned checkpoints will lead to increased checkpoints sizes. On the upside, **checkpointing times are heavily reduced**, so users will see more progress (even in unstable environments) as more up-to-date checkpoints will lighten the recovery process. You can learn more about the current limitations of unaligned checkpoints in the [documentation]({{ site.DOCS_BASE_URL }}flink-docs-release-1.11/concepts/stateful-stream-processing.html#unaligned-checkpointing), and track the improvement work planned for this feature in [FLINK-14551](https://issues.apache.org/jira/browse/FLINK-14551). 
```

Comment:
Is the unaligned option available for savepoints? If not, this may be worth mentioning since users still won't be able to rescale a job to address backlog. @pnowojski

Reply:
It is documented https://github.com/apache/flink/pull/12722 , but I'm fine if you, @morsapaes and @AHeise think it's worth mentioning here as well.

Reply:
I'm updating the link to the limitations to a section of the docs that is short and clear about it, including the issue with savepoints.